build.sbt

```diff
@@ -0,0 +1,10 @@
+import Settings._
+
+lazy val root = project.in(file("automation"))
+  .settings(rootSettings:_*)
+
+name := "root"
+
+version := "1.0"
+
+scalaVersion := "2.11.8"
```

Comment:
Just copied from other FireCloud projects. I guess this would be a good time to make a move toward 2.12.